src/Tasks.UnitTests/Copy_Tests.cs

```diff
@@ -2075,10 +2095,10 @@ public void CopyWithHardAndSymbolicLinks()
 
                 bool success = t.Execute();
 
-                Assert.False(success);
-
+                Assert.True(success);
+                me.AssertLogContains("0x80070011");
```

Comment:
What is this? At minimum, it deserves a comment.

Reply:
Here is output "The system cannot move the file to a different disk drive. (0x80070011)" when failed to create a hardLink in different drivers. I wil leave a comment here.